lms/views/lti/oidc.py

```diff
@@ -44,7 +44,7 @@ class OIDCRequestSchema(PyramidRequestSchema):
 
     target_link_uri = fields.Str(required=True)
     login_hint = fields.Str(required=True)
-    lti_message_hint = fields.Str(required=True)
+    lti_message_hint = fields.Str(required=False)
```

Comment:
This was a but, it was not required but the spec but it was provided by all LMSes we test so far:

https://www.imsglobal.org/spec/lti/v1p3#lti_message_hint-login-parameter